pkg/abaputils/abaputils.go

```diff
@@ -142,10 +142,13 @@ func HandleHTTPError(resp *http.Response, err error, message string, connectionD
 		var abapErrorResponse AbapError
 		bodyText, readError := ioutil.ReadAll(resp.Body)
 		if readError != nil {
-			return readError
+			return errors.Wrap(readError, err.Error())
```

Comment:
As far as I can see, this should have no effect, other than creating a new error instance. Why is this needed?

Reply:
You are right - I changed the behaviour.